lib/urcli-money.js

```diff
@@ -132,15 +141,24 @@ class Report {
     console.log(chalk.blue(`Earnings Report for ${startDate} to ${endDate}:`))
 
     for (let p in this.projects) {
-      let {name, id, ungradeable, passed, failed, earned} = this.projects[p]
+      let {name, id, ungradeable, passed, failed, earned, turnaround_time} = 
+          this.projects[p]
       console.log(`
       ${chalk.white(`Project: ${name} (${id}):`)}
           ${chalk.white(`Total reviewed: ${passed + failed}`)}
           ${chalk.white(`Ungradeable: ${ungradeable}`)}
           ${chalk.white(`Earned: ${earned}`)}
+          ${chalk.white(`Turnaround Time: ${turnaround_time.toFixed(2)}`)}
+          ${chalk.white(`Average Turnaround Time: ` +
+                  `${(turnaround_time/(passed+failed)).toFixed(2)}`)}
           `)
     }
+    console.log(chalk.bgBlack.white(`Total Reviewed: ${this.totalReviewed}`))
     console.log(chalk.bgBlack.white(`Total Earned: ${this.totalEarned}`))
+    console.log(chalk.bgBlack.white(`Total Turnaround Time: ` +
+                `${this.totalTurnaroundTime.toFixed(2)}`))
+    console.log(chalk.bgBlack.white(`Average Turnaround Time: ` +
```

Comment:
This is a really interesting addition. Looking forward to seeing my turnaround times 😄 .